hivelvet-backend/app/src/Actions/Labels/Edit.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Hivelvet open source platform - https://riadvice.tn/
+ *
+ * Copyright (c) 2022 RIADVICE SUARL and by respective authors (see below).
+ *
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation; either version 3.0 of the License, or (at your option) any later
+ * version.
+ *
+ * Hivelvet is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with Hivelvet; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Actions\Labels;
+
+use Actions\Base as BaseAction;
+use Enum\ResponseCode;
+use Models\Label;
+use Respect\Validation\Validator;
+use Validation\DataChecker;
+use Actions\RequirePrivilegeTrait;
+/**
+ * Class Edit.
+ */
+class Edit extends BaseAction
+{use RequirePrivilegeTrait;
```

Comment:
Use the format command. Trait usage should be into its single line.

Reply:
Couldn't find the format command for the backend, so I used an extension and made manual formatting to adapt it.
There's a format command but only for the frontend app.

Reply:
It is `hivelvet --fix`